app/javascript/components/align-and-fine-tune/AlignAndFineTune.jsx

```diff
@@ -0,0 +1,382 @@
+   * action at the 'mounted' event of this functional component (It's not actually mounted, but
+   * it mimics the same action).
+   */
+  useEffect(() => {
+    async function fetchData() {
+      await fetchDataFromAPI();
+    }
+    fetchData().then(() => {
+      setLoading(false);
+    });
+  }, []);
+
+  return (
+    <React.Fragment>
+      <div className="wrapper">
+        <TopNav centerContent={navCenterOptions} />
+        <div className="container-fluid container-wrapper">
+          <div className="row">
+            {loading ? (
+              <Loader />
+            ) : (
+              <React.Fragment>
+                {/* LEFT SIDE */}
+
+                <div className="col-lg-8 p-lg-5 pt-5">
+                  <SpineHeader
+                    domain={mapping.domain}
+                    hideMappedSpineTerms={hideMappedSpineTerms}
+                    setHideMappedSpineTerms={setHideMappedSpineTerms}
+                    mappingTerms={mappingTerms}
+                    mappedMappingTerms={mappedMappingTerms}
+                    spineTermsInputValue={spineTermsInputValue}
+                    filterSpineTermsOnChange={filterSpineTermsOnChange}
+                  />
+                  <div className="mt-5">
+                    {!loading && (
+                      <SpineTermsList
+                        terms={filteredSpineTerms}
+                        predicates={predicates}
+                        selectedMappingTerms={selectedMappingTerms}
+                        hideMappedSpineTerms={hideMappedSpineTerms}
+                        mappedTermsToSpineTerm={mappedTermsToSpineTerm}
+                        isMapped={spineTermIsMapped}
+                        origin={mapping.origin}
+                      />
+                    )}
+                  </div>
+                </div>
+
+                {/* RIGHT SIDE */}
+
+                <div className="col-lg-4 p-lg-5 pt-5 bg-col-secondary">
+                  <MappingTermsHeaders
+                    organizationName={user.organization.name}
+                    domain={mapping.domain}
+                    selectedMappingTerms={selectedMappingTerms}
+                    hideMappedMappingTerms={hideMappedMappingTerms}
+                    setHideMappedMappingTerms={setHideMappedMappingTerms}
+                    mappingTerms={mappingTerms}
+                    mappedMappingTerms={mappedMappingTerms}
+                    mappingTermsInputValue={mappingTermsInputValue}
+                    filterMappingTermsOnChange={filterMappingTermsOnChange}
+                  />
+
+                  {/* MAPPING TERMS LIST */}
+
+                  <div className="pr-5 mt-5">
+                    <AlertNotice
+                      cssClass="bg-col-primary col-background"
+                      title={
+                        mappingTerms.length +
+                        " elements have been selected from the original specification"
```

Comment:
Let's use `pluralize` in order to make sure that the word's ending is consistent with the number (here and in the other places, too).